### PR TITLE
warming up user on websocket connect event

### PR DESCRIPTION
**Pull request by eishay**
/cc @stkem @andrewconner 
/fyi @2is10 @ymatsuda

### CODE REVIEW

File: kifi-backend/modules/common/app/com/keepit/search/SearchServiceClient.scala [hunk 1]
```diff
@@ -86,6 +87,10 @@ class SearchServiceClientImpl(
     call(Search.internal.updateBrowsingHistory(userId), json)
   }
 
+  def warmUpUser(userId: Id[User]): Unit = {
+    call(Search.internal.updateBrowsingHistory(userId))
```
Comment (2is10) [inline kifi-backend/modules/common/app/com/keepit/search/SearchServiceClient.scala:91]: looks like the wrong API call

File: kifi-backend/modules/search/app/com/keepit/search/MainSearcherFactory.scala [hunk 2]
```diff
@@ -80,6 +81,8 @@ class MainSearcherFactory @Inject() (
   }
 
   def warmUp(userId: Id[User]): Seq[Future[Any]] = {
+    log.info(s"warning up $userId")
```
Comment (2is10) [inline kifi-backend/modules/search/app/com/keepit/search/MainSearcherFactory.scala:84]: warning → warming

**Comment by 2is10** [comment]
cool

**Comment by eishay** [comment]
wow, great catch. thanks @2is10 !

**Pull request merged**